package.json

```diff
@@ -3,10 +3,16 @@
   "scripts": {
     "postinstall": "bower install",
     "build": "pulp build",
-    "test": "pulp test"
```

Comment:
Let's remove these `pulp` versions and just use `spago` for both `build` and `test`.

Reply:
Could you still make the `spago` ones the default ones so we can pass the build on travis?

Reply:
Remember that as it stands, ```travis.yml``` will not build it if we do this because it still uses pulp.  I'm very happy to make this change but I'd prefer to leave editing ```travis.yml``` to you - it's such a black art. Do you still want me to go ahead?